supplementary/alica_ros1/alica_ros_proxy/src/logger/AlicaRosLogger.cpp

```diff
@@ -14,19 +14,19 @@ AlicaRosLogger::AlicaRosLogger(const Verbosity verbosity, const std::string& loc
 {
     ros::console::Level level = _verbosityRosLevelMap.at(verbosity);
 
-    std::stringstream streamName;
-    streamName << ROSCONSOLE_DEFAULT_NAME << ".[ALICA] "
-               << "[" << _localAgentId << "/" << _localAgentName << "]";
-    if (ros::console::set_logger_level(streamName.str(), level)) {
+    if (ros::console::set_logger_level(_localAgentName, level)) {
         ros::console::notifyLoggerLevelsChanged();
     }
+    ROS_INFO_STREAM_NAMED(_localAgentName, "Instantiated ROS logging for ALICA");
 }
 
 void AlicaRosLogger::log(const std::string& msg, const Verbosity verbosity, const std::string& logSpace)
 {
     std::stringstream streamName;
-    streamName << "[ALICA] "
-               << "[" << _localAgentId << "/" << _localAgentName << "]";
+    streamName << _localAgentName;
+    if (!logSpace.empty()) {
+        streamName << "." << logSpace;
+    }
```

Comment:
should it be space after this part? in previous version `]` was like a delimiter

Reply:
This is matching ros namespace format

Reply:
I meant after the name, after this if block technically

Reply:
Discussed offline